src/DGtal/geometry/tools/Hull2DHelpers.ih

```diff
@@ -361,8 +362,8 @@ namespace DGtal
             }
           return 0.0;
           }
-        
-        while(getAngle(aConvexHull[i], aConvexHull[i+1], aConvexHull[j], aConvexHull[j+1]) < M_PI ){
+        while(getAngle(aConvexHull[i], aConvexHull[i+1],
+                       aConvexHull[j], aConvexHull[j+1]) < M_PI - angleTolerance ){
```

Comment:
why not use a exact geometrical predicate for convex hull computation? If I remember correctly, @troussil has got a collection of 2D predicates already available in DGtal

Reply:
Mainly for the use for alpha-thick segment, If I well remember.

Reply:
Shouldn't we fix this cvx computation with exact computations and not angle comparisons ? 
(for 1.3 or maybe the next one)

Reply:
for non integer input the tolerance is probably useful, else perhaps waiting after 1.3 could let more time

Reply:
ok could you please add an issue on this? Just to remember that this should be fixed.

Reply:
yes I add it